### ISSUE TITLE
how to train on VOC dataset？

### ISSUE THREAD

**Issue by infinite013** [WongKinYiu/ScaledYOLOv4#8]

**Comment by WongKinYiu** [comment]
Example of YOLOv4-large:
1. Follow [Training YOLO on VOC](https://pjreddie.com/darknet/yolo/) to prepare VOC data.
2. Follow [coco.data](https://github.com/WongKinYiu/ScaledYOLOv4/blob/yolov4-large/data/coco.yaml) to create voc.data.
3. add argument `--data data/voc.data` when training.

**Comment by SpongeBab** [comment]
@WongKinYiu @infinite013 
Now I can confirm @WongKinYiu  are wrong.
The YOLO format is not same as before(it also mean is not the same as yolov4 )
(Yolo-CSP copy the code of YOLOv5 of the https://github.com/ultralytics/yolov5.and it is the author of yolov3. It's has the different dataset format :)
now:
> Each row is class x_center y_center width height format.

The https://pjreddie.com/darknet/yolo/ make the VOC dataset with:

` <object-class> <x> <y> <width> <height>`

Although it can train and no error.
But it is wrong as i show at https://github.com/WongKinYiu/ScaledYOLOv4/issues/205

`x_center y_center width height` format. is a relavtive coordinate.
`<x> <y> <width> <height> `is the absolute coordinate.

**Comment by WongKinYiu** [comment]
no, the https://pjreddie.com/darknet/yolo/ make the VOC dataset with relative coordinate.

https://github.com/AlexeyAB/darknet/blob/master/scripts/voc_label.py#L12-L23

**Comment by SpongeBab** [comment]
@WongKinYiu 
The https://pjreddie.com/darknet/yolo/ make the VOC dataset with relative coordinate. It does.
But when the image is flipped, the angle is changed, and various mosaic enhancements are performed, the values ​​of x and y do not seem to change correctly. Just like the train_batch0 picture shows.

-------------
I'm wrong.I think this is a kind of mosaic enhancements.I trained 300 and get good detections.
